packages/cardano-services/src/StakePoolSearch/DbSyncStakePoolSearchProvider/DbSyncStakePoolSearch.ts

```diff
@@ -128,7 +128,7 @@ export class DbSyncStakePoolSearchProvider implements StakePoolSearchProvider {
     return { id: { isPrimary: true, name: 'pools_by_status' }, query };
   }
   private buildPoolsByPledgeMetQuery(pledgeMet: boolean) {
-    const subQueries = poolsByPledgeMetSubqueries;
+    const subQueries = [...poolsByPledgeMetSubqueries];
```

Comment:
It would be good to add readonly to it:
export const poolsByPledgeMetSubqueries: readonly SubQuery[] = [

Reply:
Even better: `export const poolsByPledgeMetSubqueries: readonly ReadonlyArray<SubQuery> = [`